language/move-prover/bytecode-to-boogie/test_mvir/verify-vector.mvir

```diff
@@ -1,27 +1,284 @@
 module VerifyVector {
     import 0x0.Vector;
 
-    public test_empty1() : Vector.T<u64> * Vector.T<u64>
+	// succeeds. [] == [].
+	public test_empty1() : Vector.T<u64> * Vector.T<u64>
     ensures RET(0) == RET(1)
     {
-	let ev1: Vector.T<u64>;
-	let ev2: Vector.T<u64>;
-	ev1 = Vector.empty<u64>();
-	ev2 = Vector.empty<u64>();
-	return (move(ev1), move(ev2));
+		let ev1: Vector.T<u64>;
+		let ev2: Vector.T<u64>;
+		ev1 = Vector.empty<u64>();
+		ev2 = Vector.empty<u64>();
+		return (move(ev1), move(ev2));
     }
 
-
+	//succeeds. [] == [].
     public test_empty2() : Vector.T<u64> * Vector.T<u64>
     ensures RET(0) == RET(1)
     {
-	let ev1: Vector.T<u64>;
-	let ev2: Vector.T<u64>;
-	let x: u64;
-	ev1 = Vector.empty<u64>();
-	ev2 = Vector.empty<u64>();
-	Vector.push_back<u64>(&mut ev1, 1);
-	x = Vector.pop_back<u64>(&mut ev1);
-	return (move(ev1), move(ev2));
+		let ev1: Vector.T<u64>;
+		let ev2: Vector.T<u64>;
+		let x: u64;
+		ev1 = Vector.empty<u64>();
+		ev2 = Vector.empty<u64>();
+		Vector.push_back<u64>(&mut ev1, 1);
+		x = Vector.pop_back<u64>(&mut ev1);
+		return (move(ev1), move(ev2));
+    }
+
+	// succeeds. [1] == [1]
+	public test_empty3() : Vector.T<u64> * Vector.T<u64>
```

Comment:
This should not be named test_empty any longer because it does not test for emptiness. I suggest to use better names then you may also not need the comment `// succeeds`.